requirements/dev.txt

```diff
@@ -121,21 +105,15 @@ pip-tools==7.3.0 \
 platformdirs==4.0.0 \
     --hash=sha256:118c954d7e949b35437270383a3f2531e99dd93cf7ce4dc8340d3356d30f173b \
     --hash=sha256:cb633b2bcf10c51af60beb0ab06d2f1d69064b43abf4c185ca6b28865f3f9731
-    # via
-    #   pylint
-    #   virtualenv
+    # via virtualenv
 pluggy==1.3.0 \
     --hash=sha256:cf61ae8f126ac6f7c451172cf30e3e43d3ca77615509771b3a984a0730651e12 \
     --hash=sha256:d89c696a773f8bd377d18e5ecda92b7a3793cbe66c87060a6fb58c7b6e1061f7
     # via pytest
-pre-commit==3.5.0 \
-    --hash=sha256:5804465c675b659b0862f07907f96295d490822a450c4c40e747d0b1c6ebcb32 \
-    --hash=sha256:841dc9aef25daba9a0238cd27984041fa0467b4199fc4852e27950664919f660
+pre-commit==3.6.0 \
+    --hash=sha256:c255039ef399049a5544b6ce13d135caba8f2c28c3b4033277a788f434308376 \
+    --hash=sha256:d30bad9abf165f7785c15a21a1f46da7d0677cb00ee7ff4c579fd38922efe15d
     # via -r requirements/dev.in
-pylint==3.0.2 \
-    --hash=sha256:0d4c286ef6d2f66c8bfb527a7f8a629009e42c99707dec821a03e1b51a4c1496 \
-    --hash=sha256:60ed5f3a9ff8b61839ff0348b3624ceeb9e6c2a92c514d81c9cc273da3b6bcda
-    # via pylint-pytest (pyproject.toml)
```

Comment:
Hello @anis-campos!

I will need your help with this dependabot-MR.

For _some reason_, dependabot decided to "trash away" the pylint installation - leading to the pylint-action failing 😅 

Have you had any experience with such an issue?

Reply:
Sorry for the delay. 

After looking into it, my best guess is described in this commit ( I committed on master to test, I can open branch if we go forward ) : https://github.com/anis-campos/pylint-pytest/commit/8da3ef43afd2de98d7bffb09e02860f0dbc84be1

As you can see here, after a rebase, everything is back to normal https://github.com/anis-campos/pylint-pytest/pull/2/files

Reply:
No worries

Your solution was "much simpler" and much more working than whatever I tried 😅 

Submit a PR, let's fix this too!

(WDYT? `pylint>=3` or `pylint`? We did say that `# For linting purposes, only pylint>3 is supported` 😅)

Reply:
`pylint` should be enough as what matters is the version defined in the `.txt` file (unlike in the previous version, without a "frozen" version, we need to properly constraint it to have the proper resolution).

Adding `=>3` would only make sense if `<4` is added. 

But that being said, it is not the purpose of this MR, so I just c/p the original lines.

Reply:
The MR https://github.com/pylint-dev/pylint-pytest/pull/39